src/api.ts

```diff
@@ -277,7 +277,7 @@ function checkQueryParamAllowed(
  * @param options Options for configuring the REST API, such as permissions, transformers, and hooks.
  */
 export function fernsRouter<T>(
-  baseModel: Model<T>,
+  baseModel: PassportLocalModel<T>,
```

Comment:
Is this right? I'd think only User would be PassportLocalModel.

Reply:
I think you are right, there is still some weirdness there but returned to Model. By using our own types, it gets around this issue.